.github/ISSUE_TEMPLATE/test-failures.yml

```diff
@@ -0,0 +1,59 @@
+name: 📑 Test failure report
+description: Issue Template for Reporting Testing Failures.
+title: "[Test Case ID]: <DESCRIPTIVE TITLE>"
+labels: ["release testing"]
```

Comment:
One note is that people who don't have write or triage access to this repo won't have the label apply. It's really annoying. You can add a GitHub Action to apply the label once the ticket is created, by looking for text that appears in the issue template: https://github.com/openedx/axim-engineering/blob/main/.github/workflows/add-GHrequest-to-team-board.yml#L25-L36

If you like I can make a PR to this branch adding the workflow, but it's also not too terribly tricky.

Reply:
I think we already have it set up: https://github.com/openedx/wg-build-test-release/issues/278#issuecomment-1528712173, is that the same workflow?

Reply:
That adds a label on a comment. What your line of code here is doing is wanting it to add the label `release testing` when the issue is _submitted_, not on a comment.

Unfortunately if someone without write access submits a ticket, this line of code in the template does nothing.

You need the GitHub action I posted if you want to add the `release testing` label whenever someone submits a ticket via this template.

Reply:
That's interesting. I added the workflow: https://github.com/openedx/wg-build-test-release/pull/281/commits/7a2dcc46181f1705a23609f2f7aeb6302c498b36
I removed one of the conditionals, the one checking for the name.

Reply:
It works:
![image](https://github.com/openedx/wg-build-test-release/assets/64440265/ca2144a8-2f4a-4243-855b-62dac092b177)